### DISCUSSION TITLE
ChatFilter

### DISCUSSION THREAD

**Discussion by TheMartyCZ** [Zrips/CMI#6341] [Feature Requests]
I would like to add a whitelist for words that chatfilter does not check.

**Comment by dkalaxdk** [comment]
Already a feature.
Check chatfilter.yml, search for whitelist.

**Comment by TheMartyCZ** [comment]
But i need swearing whitelist.

**Reply by dkalaxdk** [reply]
You can just add any words to that list.
Does not have to be URLS etc, can be any words.
Just add the regex match for it.

(If you don't do much in regex, [CMIRegexGenerator]( https://dkalaxdk.github.io/CMIRegexFilter/RegexGenerator) )

**Reply by TheMartyCZ** [reply]
Link not working, but thank you.

**Reply by dkalaxdk** [reply]
This one should: [Generator](https://dkalaxdk.github.io/CMIRegexFilter/)
